README.md

```diff
@@ -1493,6 +1493,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Pratik Mahajan](https://github.com/Pratik-Mahajan257)
   - [Pratik Sabata](https://github.com/pratiksabat/pratiksabat)
   - [Pratish Shrivastava](https://github.com/PratishShrivastava)
+  - [Pratik Sonawane](https://github.com/Rayquaza6102)
```

Comment:
Your name must be above Pratish Shrivastava.
Let us know if you need any help with that. 🙂